src/main/java/org/jetbrains/research/refactorinsight/utils/Utils.java

```diff
@@ -185,20 +185,18 @@ public static RefactoringInfo getMainRefactoringInfo(List<RefactoringInfo> infos
     if (infos.stream().anyMatch(ofType(RENAME_ATTRIBUTE))
         && infos.stream().anyMatch(ofType(CHANGE_ATTRIBUTE_TYPE))) {
       info = infos.stream().filter(ofType(RENAME_ATTRIBUTE)).findFirst().get();
-      info.setName(RefactorInsightBundle.message("change.rename.attribute"));
+      info.setType(RefactoringType.RENAME_AND_CHANGE_ATTRIBUTE_TYPE);
     } else if (infos.stream().anyMatch(ofType(RENAME_ATTRIBUTE))) {
       info = infos.stream().filter(ofType(RENAME_ATTRIBUTE)).findFirst().get();
-      info.setName(RefactorInsightBundle.message("rename.attribute"));
```

Comment:
Why this line is removed?

Reply:
Because there is no setName anymore. The name is inferred from the type, and here they are the same.